spec/monkeys_spec.rb

```diff
@@ -16,4 +16,36 @@
       expect(left.elementwise_division(right)).to eq(Matrix[[1,1,1],[1,1,1],[1,1,1]])
     end
   end
+
+  describe '#daru_vector' do
+    it 'converts Array' do
+      expect([1,2,3].daru_vector).to eq Daru::Vector.new [1,2,3]
+      expect([1,2,3].daru_vector('test', [:a, :b, :c])).to eq \
+        Daru::Vector.new [1,2,3], name: 'test', index: [:a, :b, :c]
+    end
+
+    it 'converts Range' do
+      expect((1..3).daru_vector).to eq Daru::Vector.new [1,2,3]
+      expect((1..3).daru_vector('test', [:a, :b, :c])).to eq \
+        Daru::Vector.new [1,2,3], name: 'test', index: [:a, :b, :c]
+    end
+
+    it 'converts Hash' do
+      # FIXME: is it most useful way of converting hashes?..
+      # I'd prefer something like
+      #   expect({a: 1, b: 2, c: 3}.daru_vector('test')).to eq Daru::Vector.new [1,2,3], name: 'test', index: [:a, :b, :c]
```

Comment:
That also works. It can be an additional functionality in case there are multiple keys and their values are single elements.